app/shared/ui-wizard/src/commonMain/kotlin/ui/wizard/step/ConfigureProxy.kt

```diff
@@ -0,0 +1,376 @@
+import me.him188.ani.app.ui.wizard.WizardLayoutParams
+
+@Immutable
+enum class ProxyTestCaseState {
+    INIT,
+    RUNNING,
+    SUCCESS,
+    FAILED
+}
+
+@Stable
+class ProxyTestItem(
+    val case: ProxyTestCase,
+    val state: ProxyTestCaseState,
+)
+
+@Stable
+class ProxyTestState(
+    val testRunning: Boolean,
+    val items: List<ProxyTestItem>
+) {
+    companion object {
+        @Stable
+        val Default = ProxyTestState(false, emptyList())
+    }
+}
+
+@Composable
+private fun SettingsScope.ProxyTestStatusGroup(
+    testRunning: Boolean,
+    currentTestMode: ProxyMode,
+    items: List<ProxyTestItem>,
+    onRequestReTest: () -> Unit,
+    modifier: Modifier = Modifier,
+    content: @Composable ColumnScope.() -> Unit
+) {
+    val text = remember(testRunning, currentTestMode, items) {
+        if (testRunning) {
+            "正在检测连接，请稍后"
+        } else if (items.any { it.state == ProxyTestCaseState.FAILED }) {
+            if (currentTestMode == ProxyMode.DISABLED) {
+                "部分服务连接失败，请考虑启用代理"
+            } else {
+                "部分服务连接失败，请更换代理模式或代理地址"
+            }
+        } else {
+            "所有服务连接正常"
+        }
+    }
+    val useErrorColor = remember(testRunning, items) {
+        !testRunning && items.any { it.state == ProxyTestCaseState.FAILED }
+    }
+
+    Group(
+        title = {
+            Text(
+                text = text,
+                color = if (useErrorColor)
+                    MaterialTheme.colorScheme.error else MaterialTheme.colorScheme.onSurface,
+            )
+        },
+        actions = if (items.any { it.state == ProxyTestCaseState.FAILED }) {
+            {
+                TextButton(onRequestReTest) {
+                    Text("重新测试")
+                }
+            }
+        } else null,
+        useThinHeader = true,
+        modifier = modifier,
+        content = content,
+    )
+}
+
+@Composable
+private fun renderTestCaseName(case: ProxyTestCase): String {
+    return when (case.name) {
+        ProxyTestCaseEnums.ANI_DANMAKU_API -> "Animeko"
+        ProxyTestCaseEnums.BANGUMI_V0 -> "Bangumi"
+        ProxyTestCaseEnums.BANGUMI_P1 -> "Bangumi"
+    }
+}
+
+@Composable
+private fun renderTestCaseDescription(case: ProxyTestCase): String {
+    return when (case.name) {
+        ProxyTestCaseEnums.ANI_DANMAKU_API -> "弹幕服务"
+        ProxyTestCaseEnums.BANGUMI_V0 -> "收藏数据服务"
+        ProxyTestCaseEnums.BANGUMI_P1 -> "评论服务"
+    }
+}
+
+@Composable
+private fun ProxyTestStatusIcon(
+    state: ProxyTestCaseState,
+    modifier: Modifier = Modifier,
+) {
+    Box(modifier) {
+        when (state) {
+            ProxyTestCaseState.INIT, ProxyTestCaseState.RUNNING ->
+                CircularProgressIndicator(modifier = Modifier.size(24.dp))
+
+            ProxyTestCaseState.SUCCESS ->
+                ProvideContentColor(MaterialTheme.colorScheme.onSurfaceVariant) {
+                    Icon(Icons.Default.Check, null)
```

Comment:
missing content description

```suggestion
                    Icon(Icons.Default.Check, "连接成功")
```